package.nls.json

```diff
@@ -166,6 +167,14 @@
     "AWS.command.ecr.createRepository": "Create Repository...",
     "AWS.command.ecr.deleteRepository": "Delete Repository...",
     "AWS.command.ecr.deleteTag": "Delete Tag...",
+    "AWS.command.runCommandInContainer": "Run command in container",
+    "AWS.command.ecs.runCommandInContainer.warnBeforeExecute": "Command may modify the running container {0}. Are you sure?",
+    "AWS.command.ecs.runCommandInContainer.yesDoNotShowAgain": "Yes, and don't ask again",
+    "AWS.command.ecs.runCommandInContainer.noCliFound": "Please install the AWS CLI before proceding",
+    "AWS.command.ecs.runCommandInContainer.noPluginFound": "Please install the SSM Session Manager plugin before proceding",
+    "AWS.command.ecs.runCommandInContainer.noTasks": "There are no running tasks for the service: {0}",
+    "AWS.command.ecs.runCommandInContainer.prompt": "Enter the command to run in container",
```

Comment:
Nit: "Enter the command to run in container {0}" (with the param being the container's name)?

Reply:
added suggestion